modules/FIC_FileHandler.py

```diff
@@ -73,24 +74,42 @@ def _check_repo_files(self):
             # Check all Github files exist for each repository.
             for key in files_to_check["Github"].keys():
                 if not os.path.exists(os.path.join(self.path_level, CHANGELOG_REPO_PATH, key.lower() + ".json")):
-                    self._missing_files.append(key.lower() + ".json")
+                    self._missing_files.append((key.lower() + ".json", "git"))
```

Comment:
Do we really need to double the parenthesis??

Reply:
Same question for the other ones below.

Reply:
Good question! 
The reason why we need the double parenthesis is because the ``` append ```method expects only one argument. In order to give it more than one we append a tuple. 
Without the double parenthesis we would get an error like: 
```python
TypeError: append() takes exactly one argument (2 given)
```